### PR TITLE
[MiTele] Fix (supports more urls)

### PR DESCRIPTION
**Pull request by zurfyx**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Extractor works fine, but it should also support other routes which work exactly the same way (i.e. mitele.es/series-online/la-que-se-avecina/)

### CODE REVIEW

File: youtube_dl/extractor/mitele.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def _get_player_info(self, url, webpage):
 
 class MiTeleIE(InfoExtractor):
     IE_DESC = 'mitele.es'
-    _VALID_URL = r'https?://(?:www\.)?mitele\.es/programas-tv/(?:[^/]+/)(?P<id>[^/]+)/player'
+    _VALID_URL = r'https?://(?:www\.)?mitele\.es/(?:[^/]+/)(?:[^/]+/)(?P<id>[^/]+)/player'
```
Comment (dstftw) [inline youtube_dl/extractor/mitele.py]: What's the point of this group if you don't capture it anyway?

File: youtube_dl/extractor/mitele.py [hunk 2]
```diff
@@ -109,6 +109,20 @@ class MiTeleIE(InfoExtractor):
...
+            'ext': 'mp4',
+            'title': 'Una poliza premium, un novio tirita y un mayorista con el género a la vista',
+            'description': 'Gabi, amigo y antiguo compa\xf1ero de universidad de Javi, regresa a Mirador de Montepinar. Cuando conoce a Judith inician un t\xf3rrido romance. Cansada de encadenar un fracaso sentimental con otro, la psic\xf3loga comienza a salir con el joven para tratar de enderezar su vida.',
+            'series': 'La Que Se Avecina',
+            'season': 'Temporada 9',
+            'episode': 'Capítulo 124',
+            'thumbnail': 're:(?i)^https?://.*\.jpg$',
+            'duration': 5206,
+        },
+        'add_ie': ['Ooyala'],
```
Comment (dstftw) [inline youtube_dl/extractor/mitele.py]: This test should be `only_matching`.

**Comment by zurfyx** [comment]
Did I miss anything in the second commit?

**Comment by yan12125** [comment]
@dstftw has pushed to to master: https://github.com/rg3/youtube-dl/commits/master

**Pull request closed without merging**